.github/workflows/unit-tests.yml

```
@@ -8,7 +8,7 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
-        python-version: [ 3.8 ]
+        python-version: [ "3.10" ]
         os: [ ubuntu-20.04 ]
```

Perhaps we should update these to use the latest version of Ubuntu:

```suggestion
        os: [ ubuntu-latest ]
```